### PR TITLE
Add input vars and output vars.

### PR DESCRIPTION
**Pull request by CPWstatic**

### CODE REVIEW

File: src/planner/PlanNode.h [hunk 1]
```diff
@@ -111,11 +111,20 @@ class PlanNode {
...
-    std::string varName() const {
-        return outputVar_;
+    std::string outputVar() const {
+        return outputVars_.back();
+    }
+
+    std::string outputVar(size_t index) const {
+        DCHECK_LT(index, outputVars_.size());
+        return outputVars_[index];
+    }
```
Comment (yixinglu) [inline src/planner/PlanNode.h:124]: you can use single `outputVar` function with default parameter `index=0` to keep compatible with old codes.

**Review by yixinglu — COMMENTED** [review]
others LGTM

File: src/planner/PlanNode.h (hunk 1 above)
Comment (dutor) [reply src/planner/PlanNode.h:124]: If so there will be a `if`.

**Review by dutor — APPROVED** [review]
LGTM.

**Review by dutor — APPROVED** [review]
LGTM.

**Review by nevermore3 — APPROVED** [review]

**Pull request merged**